Package.swift

```diff
@@ -78,7 +77,31 @@ let package = Package(
         ),
         .testTarget(
             name: "GoodExtensionsTests",
-            dependencies: ["GoodExtensions"]
+            dependencies: ["GoodExtensions", "GRCompatible"]
+        ),
+        .testTarget(
+            name: "GoodCacheTests",
+            dependencies: ["CombineExt", "GoodCache", "GRCompatible"]
```

Comment:
Ani tu by podla mna nemuselo byt GRCompatible. Samotna kniznica to pouziva, pridem mi to duplicitne.